src/components/Gradebook/index.jsx

```diff
@@ -13,12 +13,15 @@ import {
   Table,
   Tabs,
 } from '@edx/paragon';
+import { trackEvent } from '@redux-beacon/segment';
 import queryString from 'query-string';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDownload, faSpinner } from '@fortawesome/free-solid-svg-icons';
 import { configuration } from '../../config';
 import PageButtons from '../PageButtons';
 import { formatDateForDisplay } from '../../data/actions/utils';
+import { trackingCategory } from '../../data/store';
```

Comment:
It's not super appropriate for component code to depend directly on the store

Could you move the tracking category into a separate module by itself, to be imported in both places where it's needed?